docs/getting-started/hello-world.md

```diff
@@ -21,6 +21,8 @@ In your file, put the following code:
 --8<-- "hello-world-main.pony"
 ```
 
+[:fontawesome-solid-play: Run in playground](https://playground.ponylang.io/?snippet=hello-world-main.pony){ .md-button .md-button--primary target="_blank" }
```

Comment:
Is there any way we could automatically render this button for every `.pony` snippet instead of needing to paste this button code ourselves all over the markdown, and maintain that?

Reply:
Yes and no. Not with the current setup. But that's why I'm experimenting with #555.